packages/terra-demographics-banner/src/DemographicsBanner.jsx

```diff
@@ -0,0 +1,112 @@
+/* eslint-disable global-require, import/no-dynamic-require, react/no-unused-prop-types */
+import React, { PropTypes } from 'react';
+import i18nLoader from 'terra-i18n/lib/i18nLoader';
+import I18nProvider from 'terra-i18n/lib/I18nProvider';
+
+import './DemographicsBanner.scss';
+import DemographicsBannerDisplay from './DemographicsBannerDisplay';
+
+const propTypes = {
+  /**
+   * Additional Details to display in the banner.
+   */
+  additionalDetails: PropTypes.node,
+  /**
+   * Age of the person.
+   */
+  age: PropTypes.string,
+  /**
+   * Additional rows of key value pairs to add to the demographics banner. The rows will be added after the person rows.
+   */
+  applicationRows: PropTypes.array,
+  /**
+   * The persons date of birth
+   */
+  dateOfBirth: PropTypes.string,
+  /**
+   * The person's deceased date. Will display the banner as deceased if this value is provided
+   */
+  deceasedDate: PropTypes.string,
+  /**
+   * Gender of the Person
+   */
+  gender: PropTypes.string,
+  /**
+   * The persons gestational age.
+   */
+  gestationalAge: PropTypes.string,
+  /**
+   * Additional key value identifiers of a person's demographic information
+   */
+  identifiers: PropTypes.object,
+  /**
+   * Locale to display the banner with
+   */
+  locale: PropTypes.string,
+  /**
+   * Full Name of the person
+   */
+  personName: PropTypes.string,
+  /**
+   * Photo to display in the banner
+   */
+  photo: PropTypes.node,
+  /**
+   * The column layout in which to present the displays.
+   */
+  postMenstrualAge: PropTypes.string,
+  /**
+   * The persons preferred first name if they have one.
+   */
+  preferredFirstName: PropTypes.string,
+};
+
+const defaultProps = {
+  additionalDetails: null,
+  age: null,
+  applicationRows: [],
+  dateOfBirth: null,
+  deceasedDate: null,
+  gender: null,
+  gestationalAge: null,
+  identifiers: {},
+  locale: 'en',
```

Comment:
This is dangerous to include as a default.  If a locale is provided in the application, but not provided for this component, this component will default to `en` and we may have a mismatch in locale.